content/geometry/object-human/human-structure.adoc

```diff
@@ -8,36 +8,36 @@ legend
 |_ [[../geometry/object-human/human-index.html#_grp_root Grp_Root]]
   |_ [[../geometry/object-human/human-index.html#_armature Armature]]
     |_ [[../geometry/object-human/human-index.html#_root Root]]
-      |_ [[../geometry/object-human/human-index.html#_hip Hip]]
-        |_ [[../geometry/object-human/human-index.html#_lower_spine Lower_Spine]]
-          |_ [[../geometry/object-human/human-index.html#_upper_spine Upper_Spine]]
-            |_ [[../geometry/object-human/human-index.html#_neck Neck]]
-              |_ [[../geometry/object-human/human-index.html#_head Head]]
-                |_ [[../geometry/object-human/human-index.html#_eye_left Eye_Left]]
-                |_ [[../geometry/object-human/human-index.html#_eye_right Eye_Right]]
-          |_ [[../geometry/object-human/human-index.html#_shoulder_left Shoulder_Left]]
-            |_ [[../geometry/object-human/human-index.html#_upper_arm_left Upper_Arm_Left]]
-              |_ [[../geometry/object-human/human-index.html#_lower_arm_left Lower_Arm_Left]]
-                |_ [[../geometry/object-human/human-index.html#_hand_left Hand_Left]]
-                  |_ [[../geometry/object-human/human-index.html#_full_lingers_left Full_Fingers_Left]]
-                  |_ [[../geometry/object-human/human-index.html#_full_thumb_left Full_Thumb_Left]]
-            |_ [[../geometry/object-human/human-index.html#_upper_arm_right Upper_Arm_Right]]
-              |_ [[../geometry/object-human/human-index.html#_lower_arm_right Lower_Arm_Right]]
-                |_ [[../geometry/object-human/human-index.html#_hand_right Hand_Right]]
-                  |_ [[../geometry/object-human/human-index.html#_full_fingers_right Full_Fingers_Right]]
-                  |_ [[../geometry/object-human/human-index.html#_full_thumb_right Full_Thumb_Right]]
-      |_ [[../geometry/object-human/human-index.html#_upper_leg_left Upper_Leg_Left]]
-        |_ [[../geometry/object-human/human-index.html#_lower_leg_left Lower_Leg_Left]]
-          |_ [[../geometry/object-human/human-index.html#_foot_left Foot_Left]]
-            |_ [[../geometry/object-human/human-index.html#_full_toes_left Full_Toes_Left]]
-      |_ [[../geometry/object-human/human-index.html#_upper_leg_right Upper_Leg_Right]]
-        |_ [[../geometry/object-human/human-index.html#_lower_leg_right Lower_Leg_Right]]
-          |_ [[../geometry/object-human/human-index.html#_foot_right Foot_Right]]
-            |_ [[../geometry/object-human/human-index.html#_full_toes_right Full_Toes_Right]]
-    |_ [[../geometry/object-human/human-index.html#_accessories Accessories_Wristband]]
-    |_ [[../geometry/object-human/human-index.html#_body Body_Human]]
-    |_ [[../geometry/object-human/human-index.html#_clothing Clothing_Vest]]
-    |_ [[../geometry/object-human/human-index.html#_hair Hair_Short]]
+      |_ [[../geometry/object-human/human-index.html#_hip Hip]] (T)
```

Comment:
You need to add _t to all links where you added the (T) in the headline, e.g. ../geometry/object-human/human-index.html#_hip_t. Otherwise the link won't work.

Adding the (T) outside of the link display name is a very good idea. We should do that in vehicle structure and environment structure as well. Would you mind doing that in this PR directly? I don't think we need another issue for this.